src/api/index.js

```diff
@@ -140,3 +146,89 @@ export const updateGroup = ({ uuid, systemIDs, groupName }) => {
 export const fetchActiveImages = () => {
   return instance.get('/api/image-builder/v1/composes');
 };
+
+export const fetchDeviceSummary = async () => {
+  const client = new HostsApi(undefined, '/api/inventory/v1/', instance);
+  return await Promise.all([
+    client.apiHostGetHostList(
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      ['fresh'],
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      {
+        query: generateFilter(inventoryFields, 'fields'),
```

Comment:
No need for the `fields` in here. It should be just the default `filters`
```suggestion
        query: generateFilter(inventoryFields),
```